Automattic/wp-calypso
# Signup: add site preview in the onboarding flow

### Changes proposed in this Pull Request

Adds a `SiteMockup` component to the `onboarding-dev` flow for later inclusion in the `onboarding` flow. This will allow users to see a stylized mockup of how the information will be displayed in the Radcliffe 2 theme once they complete signup.

**Note** This is not yet hooked up to the verticals API endpoint, which will supply 

### Testing instructions

1. Enter the `onboarding-dev` flow via `/start/onboarding-dev/`
2. On a wide screen, both the desktop and mobile mockups should appear beneath the step content
3. On a small screen, only the mobile mockup should appear
4. In the `site-topic` step, choose "Mexican Restaurant" to move from outlines to seeing site mockup content
5. In the `site-information` step (after choosing **business** in the `site-type` step), you should see the name, address, and phone number reflected in the mockup
6. Try to break things by not following exactly the steps above.


Wide screen
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/195089/49488144-0e1e5e00-f80b-11e8-944e-02e7a424af61.png">

Small screen
<img width="376" alt="image" src="https://user-images.githubusercontent.com/195089/49488196-3d34cf80-f80b-11e8-9ef1-4103b4952ded.png">

Site information step
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/195089/49488361-f0052d80-f80b-11e8-8658-c91459699eda.png">
